osf-model/src/main/resources/ontology/osf-model.rdf

```diff
@@ -773,6 +773,65 @@
         <rdfs:range rdf:resource="&xsd;string"/>
     </owl:DatatypeProperty>
 
+    <owl:DatatypeProperty rdf:about="&osfbom;hasContent>
+        <rdfs:label xml:lang="en">hasContent</rdfs:label>
+        <rdfs:comment xml:lang="en">The text of this comment</rdfs:comment>
+        <rdfs:domain rdf:resource="&osfbom;Comment"/>
+        <rdfs:range rdf:resource="&xsd;string"/>
+    </owl:DatatypeProperty>
+
+    <owl:DatatypeProperty rdf:about="&osfbom;isDeleted>
+        <rdfs:label xml:lang="en">isDeleted</rdfs:label>
+        <rdfs:comment xml:lang="en">Whether this comment is deleted</rdfs:comment>
+        <rdfs:domain rdf:resource="&osfbom;Comment"/>
+        <rdfs:range rdf:resource="&xsd;boolean"/>
+    </owl:DatatypeProperty>
+
+    <owl:DatatypeProperty rdf:about="&osfbom;isDeleted>
+        <rdfs:label xml:lang="en">isDeleted</rdfs:label>
+        <rdfs:comment xml:lang="en">Whether this comment is deleted</rdfs:comment>
+        <rdfs:domain rdf:resource="&osfbom;Comment"/>
+        <rdfs:range rdf:resource="&xsd;boolean"/>
+    </owl:DatatypeProperty>
+
+    <!-- "hasNode" is already taken -->
```

Comment:
however, there are other things which have `Node` as a target. E.g. `registeredFrom` should be kept since it is a different semantic meaning (it is more of a peer relationship whereas `hasNode` is more of an ownership).